x-pack/test/security_solution_cypress/package.json

```diff
@@ -16,8 +16,8 @@
     "cypress:detection_engine:exceptions:run:ess": "yarn cypress:ess --spec './cypress/e2e/detection_response/detection_engine/exceptions/**/*.cy.ts'",
```

Comment:
Will revert these changes after the flaky test runner run here: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7267

Reply:
successful!! https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7267 😃